client/state/sites/selectors/get-site-by-slug.js

```diff
@@ -19,6 +14,6 @@ import getSiteSlug from './get-site-slug';
  */
 export default createSelector(
 	( state, siteSlug ) =>
-		find( getSitesItems( state ), ( site ) => getSiteSlug( state, site.ID ) === siteSlug ) || null,
+		getSitesItems( state ).find( ( site ) => getSiteSlug( state, site.ID ) === siteSlug ) ?? null,
```

Comment:
Since `getSitesItems()` will return an object, should we wrap it in a `Object.values()` before calling `.find()` on it?

And since we actually only need the ID of the site, we could use `Object.keys()`.

Reply:
Good catch, this actually crashed Calypso at runtime. Fixing.

Reply:
> And since we actually only need the ID of the site, we could use `Object.keys()`.

In this case we need `Object.values()` because the result of `.find` is supposed to be the site object.